app/models/division.rb

```diff
@@ -50,8 +50,7 @@ def self.root_internal_name
 
   def self.root
     # create on demand if not present for convenience of blank db's and test cases
-    # @@root ||=
```

Comment:
Okay.  I'll replace the find by internal name with the closure_tree root method, add validation logic and have faith that the root will be unique.

I still want a way to fail-fast with a useful error message if for any reason the resolved root node is missing.  Essentially an "assert".  Do you have a convention / style around "assert" style checks?

Also, the root node is currently being created in seeds.rb.  I've been meaning to ask you if you still want to keep seeds.rb and if so, how much should be in there.  For my own development, I've been using lib/legacy/static_data.rb which includes the more through set of data relevant to the migrated system.  So I'm happy to be brutal in stripping down seeds.rb to the bare minimum that you think is appropriate.